packages/zowe-explorer-api/README.md

```diff
@@ -53,4 +51,4 @@ See this [special extension document](https://github.com/zowe/zowe-explorer-vsco
 
 Extensibility API for Zowe Explorer is part of the [Zowe Explorer monorepo on Github](https://github.com/zowe/zowe-explorer-vscode). You find the sources there in the `/packages/zowe-explorer-api` sub-folder.
 
-To file issues, use the [Zowe Explorer issue list](https://github.com/zowe/zowe-explorer-vscode/issues) after reviewing the [API Roadmap item](https://github.com/zowe/zowe-explorer-vscode/issues/837).
+To file issues, use the [Zowe Explorer issue list](https://github.com/zowe/zowe-explorer-vscode/issues).
```

Comment:
This seems strange when reading it, maybe it could be something like
`Bugs and Enhancement requests can be filed in the [Zowe Explorer repository](https://github.com/zowe/zowe-explorer-vscode/issues).